build.gradle

```diff
@@ -287,6 +293,8 @@ task shadowTestJar(type: ShadowJar) {
     from(project.sourceSets.main.output, project.sourceSets.test.output)
     configurations = [project.configurations.testRuntime]
     dependencies {
+        include(dependency('org.slf4j:slf4j-api:1.7.25'))
```

Comment:
Ah, great.  In that case, @bw2 can you also include the compile dependency for slf4j from the linked commit?